### ISSUE TITLE
Make a loading screen based off the R4 theme

### ISSUE THREAD

**Issue by NightYoshi370** [ahezard/nds-bootstrap#506]
So while I have spoken about this on Discord, I'd like to put it in a place where it wouldn't just scroll by after a few minutes, so here I am.

Basically, when researching about the r4 card the other day, I came across a video and in it showed the r4 loading screen:
![image](https://user-images.githubusercontent.com/18664762/51641044-05e4c400-1f1a-11e9-898d-f7f609a791b1.png)

That got me thinking: What if the loading screen could be different for r4 theme users? Here's what I mean.
Basically, what I'd like nds-bootstrap to do is load the twilightMenu++ ini (if it exists), check which theme it is and check if the loading screen is "regular". If it is, show the loading screen in a format like that. Now, obviously, I don't mean the actual bg. I mean the small bar in the middle and the loading icon right next to it.

It would be a nice addition that would be bring more accuracy to the r4 theme users.

**Comment by NightYoshi370** [comment]
Alternatively, you could make TWiLightMenu++ save the theme it uses to the nds-bootstrap ini. That way, we could do much more than *just* this.

**Comment by NightYoshi370** [comment]
![image](https://user-images.githubusercontent.com/18664762/51656983-cb9a1780-1f57-11e9-9ac3-d614f7af34ba.png)

I made an image mockup of how it can look like

**Comment by FlameKat53** [comment]
Well, I made one, exactly like your mock-up and PR-ed it
